### PR TITLE
Netmiko connection 

### PR DESCRIPTION
**Pull request by ktbyers**

### CODE REVIEW

**Comment by ktbyers** [comment]
See here for additional reference;

https://github.com/napalm-automation/brigade/pull/45

**Comment by ktbyers** [comment]
@dbarrosop Where do you want it documented?

Here is the error you will get when this happens (though should be rare):

```
AttributeError: 'str' object has no attribute 'host'
```

Basically, you will pass in a host to netmiko_connect and then you will get an error because that will be a string and not a Brigade `host object`.

**Comment by ktbyers** [comment]
Okay, updated and added a message to more clearly indicate the issue (if someone tries to incorrectly pass in a Netmiko 'host' argument).

File: brigade/plugins/tasks/connections/netmiko_connection.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    'nxos': 'cisco_nxos',
+    'eos': 'arista_eos',
+    'junos': 'juniper_junos',
+    'iosxr': 'cisco_iosxr'
+}
+
+
+def netmiko_connection(task=None, host=None, **netmiko_args):
+    """Connect to the host using Netmiko and set the relevant connection in the connection map.
+
```
Comment (dbarrosop) [inline brigade/plugins/tasks/connections/netmiko_connection.py]: Should we add a note about `netmiko_host` mapping to `host` in netmiko's ConnectHandler?

**Comment by dbarrosop** [comment]
Yeah, that's smart. The only comment I have is if it's worth adding a note to the docstring regarding that so it shows in the documentation.

**Comment by dbarrosop** [comment]
other than the previous comment +1

**Pull request merged**